src/components/transport_manager/src/transport_adapter/transport_adapter_impl.cc

```diff
@@ -46,12 +46,13 @@ namespace transport_adapter {
 CREATE_LOGGERPTR_GLOBAL(logger_, "TransportManager")
 namespace {
 DeviceTypes devicesType = {
-    std::make_pair(AOA, std::string("USB_AOA")),
-    std::make_pair(PASA_AOA, std::string("USB_AOA")),
-    std::make_pair(MME, std::string("USB_IOS")),
-    std::make_pair(BLUETOOTH, std::string("BLUETOOTH")),
-    std::make_pair(PASA_BLUETOOTH, std::string("BLUETOOTH")),
-    std::make_pair(TCP, std::string("WIFI"))};
+    std::make_pair(DeviceType::AOA, std::string("USB_AOA")),
+    std::make_pair(DeviceType::PASA_AOA, std::string("USB_AOA")),
```

Comment:
why PASA_AOA was removed and get back?

Reply:
@yang1070 
Perhaps this is old commit. I don't see these changes with latest version https://github.com/smartdevicelink/sdl_core/blob/feature/IAP_over_BT/src/components/transport_manager/src/transport_adapter/transport_adapter_impl.cc